src/stdio_mgr/stdio_mgr.py

```diff
@@ -289,17 +310,8 @@ def __exit__(self, exc_type, exc_value, traceback):
         self._close_files()
 
 
-class StdioManager(_MultiCloseContextManager):
-    r"""Substitute temporary text buffers for `stdio` in a managed context.
-
-    Context manager.
-
-    Substitutes empty :class:`RandomTextIO`\ s for
-    :obj:`sys.stdout` and :obj:`sys.stderr`,
-    and a :class:`TeeStdin` for :obj:`sys.stdin` within the managed context.
-
-    Upon exiting the context, the original stream objects are restored
-    within :mod:`sys`, and the temporary streams are closed.
+class FakeStdioTuple(StdioTuple, _MultiCloseContextManager):
```

Comment:
the `_MultiCloseContextManager` here has no effect.  It is mostly thought experiment atm, and should be removed unless there is some benefit in keeping it.

Reply:
Ah, the important part is that dual-inheritance is possible, so others could do it if they wanted to use `_MultiCloseContextManager` as a mixin.